src/spec/validitygenerator.py

```diff
@@ -508,7 +513,15 @@ def createValidationLineForParameter(self, blockname, param, params, typecategor
                 asciidoc += '\n'
             else:
                 if self.paramIsArray(param):
-                    asciidoc += self.makeAsciiDocLineForParameter(blockname, param, params, 'combinations of ' + self.makeEnumerationName(bitsname) + ' value')
+                    if param.text is not None and 'const' in param.text:
+                        asciidoc += self.makeAsciiDocLineForParameter(blockname, param, params, 'combinations of ' + self.makeEnumerationName(bitsname) + ' value')
```

Comment:
Should this one be "values" (plural) instead of "value"?

Reply:
I think there is a hack somewhere in the script adding "s" to pluralize things.

Reply:
Indeed there is. It's... a fun script.